src/vm/eepolicy.cpp

```diff
@@ -1176,18 +1176,31 @@ inline void LogCallstackForLogWorker()
 // Return Value:
 //    None
 //
-inline void DoLogForFailFastException(LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo)
+inline void DoLogForFailFastException(LPCWSTR pszMessage, PEXCEPTION_POINTERS pExceptionInfo, UINT errorSource)
 {
     WRAPPER_NO_CONTRACT;
 
     Thread *pThread = GetThread();
     EX_TRY
     {
-        PrintToStdErrA("FailFast: ");
+        if (errorSource == EEPolicy::FFES_FailFast)
+        {
+            PrintToStdErrA("FailFast: ");
+        }
+        else if (errorSource == EEPolicy::FFES_FailFast)
```

Comment:
I don't think this works : ) All the clauses are the same.

Reply:
Oops, thanks for spotting that :)